src/logics/uniswap-v3/configs.ts

```diff
@@ -1,9 +1,9 @@
 import { FeeAmount } from '@uniswap/v3-sdk';
-import { arbitrumTokens, mainnetTokens, polygonTokens } from './tokens';
+import { arbitrumTokens, avalancheTokens, baseTokens, mainnetTokens, optimismTokens, polygonTokens } from './tokens';
 import * as common from '@protocolink/common';
 import * as univ3 from 'src/modules/univ3';
 
-// https://github.com/Uniswap/interface/blob/v4.204.5/src/constants/routing.ts#L61
+// https://github.com/Uniswap/interface/blob/v4.265.0/src/constants/routing.ts#L65
```

Comment:
The native on each chain is applied as base after `v4.265.0`. Can this be updated as well?

Reply:
The native tokens were used as base tokens at `v4.204.5` as well but we didn't include them in this file. Do we need to add the native tokens here? @chouandy